cg/meta/report/templates/macros/data_analysis/qc_metrics/rnafusion_qc_metrics.html

```diff
@@ -3,7 +3,8 @@
 {% macro rnafusion_qc_metrics(samples) %}
   {% set
     metrics = [
-      {"name": "RNA Integrity Number (RIN)",                   "key": "rin",                   "description": "Mått på skala från 1 till 10 för kvalitet och integritet hos RNA-prover."},
+      {"name": "RNA Integrity Number (RIN)",                   "key": "rin",                   "description": "Mått på skala från 1 till 10 för kvalitet och integritet hos RNA-prover. N/A som ett värde är en varning att RIN kanske inte är tillförlitlig för det specifika provet."},
```

Comment:
Not reliable or not available. What about something like this (Swedish to be corrected)?
```suggestion
      {"name": "RNA Integrity Number (RIN)",                   "key": "rin",                   "description": "Mått på skala från 1 till 10 för kvalitet och integritet hos RNA-prover. N/A som värde är en varning om att inget tillförlitligt RIN är tillgängligt för det specifika det provet."},
```

Reply:
Will also double check with KN